web/src/components/CreateTestPlugins/Grpc/index.ts

```diff
@@ -1,2 +1,2 @@
 // eslint-disable-next-line no-restricted-exports
-export {default} from './Rpc';
+export {default} from './Grpc';
```

Comment:
Updating all references of RPC to GRPC

Reply:
Why GRPC? I mean the OTEL specification uses just `RPC` in the [convention.](https://github.com/open-telemetry/opentelemetry-specification/blob/6ce62202e5407518e19c56c445c13682ef51a51d/specification/metrics/semantic_conventions/rpc.md)